### ISSUE TITLE
Tensorflow version 1.xx is PAIN (COLAB)

### ISSUE THREAD

**Issue by sahilg06** [MRzzm/DINet#115]
Follow this to run inference on **COLAB**

**step-1**: Restart session with GPU

**step-2**: Install these
```
!pip install resampy
!pip install python_speech_features
```

**step-3**: Change these https://github.com/MRzzm/DINet/blob/3b57fb0a2482213327890fbb76baeafdaa412597/utils/deep_speech.py#L22-L24 to

```   
logits_ph = graph.get_tensor_by_name("logits:0")
input_node_ph = graph.get_tensor_by_name("input_node:0")
input_lengths_ph = graph.get_tensor_by_name("input_lengths:0")
```
**step-4**: Wherever np.xx (xx: data type int/float) is written, change it to xx.
1. https://github.com/MRzzm/DINet/blob/3b57fb0a2482213327890fbb76baeafdaa412597/utils/deep_speech.py#L75
2. https://github.com/MRzzm/DINet/blob/3b57fb0a2482213327890fbb76baeafdaa412597/utils/deep_speech.py#L79
3. https://github.com/MRzzm/DINet/blob/3b57fb0a2482213327890fbb76baeafdaa412597/inference.py#L54
4. https://github.com/MRzzm/DINet/blob/3b57fb0a2482213327890fbb76baeafdaa412597/utils/data_processing.py#L41

**Comment by Inferencer** [comment]
Agreed, first thing I did with [LipSick](https://github.com/Inferencer/LipSick) (DINet with some extra features) was to upgrade the versions
